File: PartnerApi/Python/bz_delete_computer.py
import json
import requests

api_url = ""  # Provided by b2_authorize_account
account_authorization_token = ""  # Provided by b2_authorize_account
account_id = ""  # Provided by b2_list_group_members
computer_id = ""  # Provided by bz_list_computers

response = requests.post(
    f"{api_url}/api/backup/v1/bz_delete_computer",
    headers={"Authorization": account_authorization_token},
    json={"accountId": account_id, "computerId": computer_id},
)
response_data = response.json()
print(json.dumps(response_data, indent=2))

File: PartnerApi/Java/BzDeleteComputer.java
package com.backblaze.examples;

import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;

public class BzDeleteComputer {
    public static void main(String[] args) {
        String apiUrl = ""; // Provided by b2_authorize_account
        String accountAuthorizationToken = ""; // Provided by b2_authorize_account
        String accountId = ""; // Provided by b2_list_group_members
        String computerId = ""; // Provided by bz_list_computers

        String postParams = "{\"accountId\":\"" + accountId + "\",\"computerId\":\"" + computerId + "\"}";
        try (HttpClient client = HttpClient.newHttpClient()) {
            HttpRequest request = HttpRequest.newBuilder()
                    .uri(URI.create(apiUrl + "/api/backup/v1/bz_delete_computer"))
                    .header("Authorization", accountAuthorizationToken)
                    .header("Content-Type", "application/json")
                    .POST(HttpRequest.BodyPublishers.ofString(postParams))
                    .build();
            HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
            String jsonResponse = response.body();
            System.out.println(jsonResponse);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
